physionet-django/user/templates/user/credential_application.html

```diff
@@ -11,16 +11,23 @@
 {% block content %}
 <div class="container">
   {% include "message_snippet.html" %}
-  <h1>PhysioNet Credentialing Application</h1>
+  <h1>Application for Credentialed Access</h1>
   <hr>
 
   <p>Please use the form below to apply for PhysioNet credentialing. In order to apply:</p>
   <ul>
     <li>Complete the CITI Program in "Data or Specimens Only Research", an online course that covers ethics of human research and patient privacy. Instructions are provided <a href="{% url 'citi_course' %}">here</a>.</li>
     <li>If you have an ORCID iD please link to it in your settings (<a href="{% url 'edit_orcid' %}">ORCID settings</a>) as this may help us expedite your application by making it easier to verify your identity.</li>
```

Comment:
Perhaps we should move this line about the applicant adding their own ORCID iD to the new timely review section below.

Reply:
I thought about moving, but ended up leaving it here because I kind of like it being a soft requirement. Lucas's [pull request](https://github.com/MIT-LCP/physionet-build/pull/1189) moves this whole section, so perhaps we can rethink when we review his PR?

Reply:
Sounds good!